### PR TITLE
solved the invalid projectId while deleting project issue#5451

### PR DESCRIPTION
**Pull request by akshay-bharadva**
Close: #5451 

## PR Details

solved the issue : Invalid `projectId` when trying to delete a project

## PR Checklist

- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

image

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by overbit — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Workspaces/ResourceList.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ function ResourceList() {
   });
 
   const handleResourceDelete = useCallback(
-    (resource) => {
+    (resource: any) => {
```
[the remark this answers is not in the document]
Comment (akshay-bharadva) [reply packages/amplication-client/src/Workspaces/ResourceList.tsx]: solved the comment

File: packages/amplication-client/src/Workspaces/ResourceList.tsx [hunk 2]
```diff
@@ -112,16 +112,22 @@ function ResourceList() {
...
-      eventName: AnalyticsEventNames.ProjectDelete,
-    });
-    deleteProject({
-      variables: {
-        projectId: currentProject.id,
-      },
-    }).catch(setError);
-  }, [currentProject, deleteProject, setError, trackEvent]);
+  const handleProjectDelete = useCallback(
+    (project) => {
```
Comment (MichaelSolati) [inline packages/amplication-client/src/Workspaces/ResourceList.tsx:116]: Is project being used anywhere?

**Review by MichaelSolati — APPROVED** [review]
I think this looks good, but would love @overbit to confirm.

Also please see question regarding `handleProjectDelete` function. If `project` arg is not used, please remove it.

**Comment by MichaelSolati** [comment]
Oh, and add the video from discord here!

File: packages/amplication-client/src/Workspaces/ResourceList.tsx (hunk 2 above)
Comment (akshay-bharadva) [reply packages/amplication-client/src/Workspaces/ResourceList.tsx:116]: No, let me remove it and upload the video that I shared in Discord.

**Comment by akshay-bharadva** [comment]
https://user-images.githubusercontent.com/52954931/226988687-941430e8-fbba-49db-aca9-85bfde621064.mp4

**Review by overbit — APPROVED** [review]

**Pull request merged**